rio/.deploy_cfg

```diff
@@ -0,0 +1,3 @@
+[auth]
+hostname = 1721
+
```

Comment:
This whole file should be ignored

Reply:
import the gitignore file please, it should be on the main branch